README-REST.md

```diff
@@ -0,0 +1,1122 @@
+
+#### Get time
+```python
+response = bitvavo.time()
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+{
+  "time": 1543397021396
+}
+```
+</details>
+
+#### Get markets
+```python
+# options: market
+response = bitvavo.markets({})
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+{
+  "market": "ADA-BTC",
+  "status": "trading",
+  "base": "ADA",
+  "quote": "BTC",
+  "pricePrecision": 5,
+  "minOrderInBaseAsset": "100",
+  "minOrderInQuoteAsset": "0.001",
+  "orderTypes": [
+    "market",
+    "limit"
+  ]
+}
+{
+  "market": "ADA-EUR",
+  "status": "trading",
+  "base": "ADA",
+  "quote": "EUR",
+  "pricePrecision": 5,
+  "minOrderInBaseAsset": "100",
+  "minOrderInQuoteAsset": "10",
+  "orderTypes": [
+    "market",
+    "limit"
+  ]
+}
+{
+  "market": "AE-BTC",
+  "status": "trading",
+  "base": "AE",
+  "quote": "BTC",
+  "pricePrecision": 5,
+  "minOrderInBaseAsset": "10",
+  "minOrderInQuoteAsset": "0.001",
+  "orderTypes": [
+    "market",
+    "limit"
+  ]
+}
+{
+  "market": "AE-EUR",
+  "status": "trading",
+  "base": "AE",
+  "quote": "EUR",
+  "pricePrecision": 5,
+  "minOrderInBaseAsset": "10",
+  "minOrderInQuoteAsset": "10",
+  "orderTypes": [
+    "market",
+    "limit"
+  ]
+}
+...
+```
+</details>
+
+#### Get assets
+```python
+# options: symbol
+response = bitvavo.assets({})
+print(response)
+```
+<details>
+ <summary>View Response</summary>
+
+```python
+[
+  {
+    "symbol": "ADA",
+    "name": "Cardano",
+    "decimals": 6,
+    "depositFee": "0",
+    "depositConfirmations": 20,
+    "depositStatus": "OK",
+    "withdrawalFee": "0.2",
+    "withdrawalMinAmount": "0.2",
+    "withdrawalStatus": "OK",
+    "networks": [
+      "Mainnet"
+    ],
+    "message": ""
+  },
+  {
+    "symbol": "AE",
+    "name": "Aeternity",
+    "decimals": 8,
+    "depositFee": "0",
+    "depositConfirmations": 30,
+    "depositStatus": "OK",
+    "withdrawalFee": "2",
+    "withdrawalMinAmount": "2",
+    "withdrawalStatus": "OK",
+    "networks": [
+      "Mainnet"
+    ],
+    "message": ""
+  },
+  {
+    "symbol": "AION",
+    "name": "Aion",
+    "decimals": 8,
+    "depositFee": "0",
+    "depositConfirmations": 0,
+    "depositStatus": "",
+    "withdrawalFee": "3",
+    "withdrawalMinAmount": "3",
+    "withdrawalStatus": "",
+    "networks": [
+      "Mainnet"
+    ],
+    "message": ""
+  },
+  {
+    "symbol": "ANT",
+    "name": "Aragon",
+    "decimals": 8,
+    "depositFee": "0",
+    "depositConfirmations": 30,
+    "depositStatus": "OK",
+    "withdrawalFee": "2",
+    "withdrawalMinAmount": "2",
+    "withdrawalStatus": "OK",
+    "networks": [
+      "Mainnet"
+    ],
+    "message": ""
+  },
+  ...
+]
+```
+</details>
+
+### Market Data
```

Comment:
:art: Maybe it's nice to add a qualifier here:
```suggestion
### Market Data Endpoints
```

Reply:
Nice, done.